.pipelines/.vsts-vhd-builder-release.yaml

```
@@ -46,6 +46,10 @@ parameters:
   displayName: Build MarinerV1 Gen2
   type: boolean
   default: true
+- name: buildMarinerV2gen1
```

```suggestion
- name: buildMarinerV2
```

Should probably drop the gen1 suffix from the changes in this file for consistency with the existing build targets.